date-adder.js

```diff
@@ -0,0 +1,17 @@
+function add(date, diff) {
```

Comment:
since you are using `setSeconds`, `setMinutes`, etc., you are changing the original date that was passed to your function. This is called a side effect, which is something that you generally want to avoid.